packages/web/components/RegisterPlayer.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button, ButtonGroup, Flex, Text } from '@metafam/ds';
+import React from 'react';
+
+import { Icon3box } from './icons/Icon3box';
+
+export const RegisterPlayer: React.FC = () => {
+  return (
+    <Flex
+      alignItems="center"
+      justifyContent="center"
+      flexDirection="column"
+      flex={1}
+    >
+      <Text
+        fontSize="2xl"
+        fontFamily="mono"
+        fontWeight="bold"
+        color="white"
+        m={5}
+      >
+        Register your Player account
+      </Text>
+      <ButtonGroup spacing={5} marginTop="20">
+        <Button
+          color="white"
+          variant="outline"
+          size="lg"
+          padding="2rem"
+          width="8rem"
+          leftIcon={Icon3box}
+          textTransform="uppercase"
+          alignItems="center"
+        >
+          box
+        </Button>
+        <Button
+          color="white"
+          variant="outline"
+          size="lg"
+          padding="2rem"
+          width="8rem"
+        >
+          GitHub
+        </Button>
+        <Button
+          color="white"
+          variant="outline"
+          size="lg"
+          padding="2rem"
+          leftIcon="email"
+          width="8rem"
```

Comment:
Instead of repeating these props for every component, we should just create our own Button component in the design system and set these props by default so that we can control it all in one place, otherwise becomes hard to refactor.